modules/sdk/project-templates/project-templates-rest/src/main/resources/archetype-resources/pom.xml

```diff
@@ -16,7 +16,7 @@
 		<dependency>
 			<groupId>javax.ws.rs</groupId>
 			<artifactId>javax.ws.rs-api</artifactId>
-			<version>2.0.1</version>
+			<version>2.1</version>
```

Comment:
Hey @petershin  i want to hold off sending to brian until we know the compatibility level of this change (which portal versions it will work on)

Reply:
I tested this using the blade sample and the bundle still activates